src/utils/rocksdb_serialization.hpp

```diff
@@ -167,10 +207,10 @@ inline std::string SerializeVertexAsValueForUniqueConstraint(const storage::Labe
 }
 
 inline storage::LabelId DeserializeConstraintLabelFromUniqueConstraintStorage(const std::string &key) {
-  std::vector<std::string> key_vector = utils::Split(key, "|");
-  std::vector<std::string> constraint_key = utils::Split(key_vector[0], ",");
+  std::string_view firstPartKey = GetViewOfFirstPartOfSplit(key, '|');
+  std::string_view constraint_key = GetViewOfFirstPartOfSplit(firstPartKey, ',');
   /// TODO: andi Change this to deserialization method directly into the LabelId class
-  return storage::LabelId::FromUint(std::stoull(constraint_key[0]));
+  return storage::LabelId::FromUint(std::stoull(std::string(constraint_key)));
```

Comment:
Way to avoid creating a string by using std::from_chars: https://stackoverflow.com/a/63125672